Adrian-Popescu/README.md

```diff
@@ -31,4 +31,4 @@ To install syft with pip, you can issue `pip install syft`.
 
 | Name | Slack |
 | ------ | ------ |
-| Adrian Popescu | Adrian Popescu |
\ No newline at end of file
+| Adrian Popescu | Adrian Popescu |
```

Comment:
You have changed another person's files can you please revert this file changes!

Reply:
Also, the folder name should be your slack name and not the name of the project

Reply:
@Shanmugapriya03 this is the group project and it is mentioned to specify group project with the name of the project

Reply:
okay I just saw that now sorry for the inconvenience but can you revert the change you made to `Adrian-Popescu/README.md`